src/Pages/Login/SearchPwd/SearchPwd.component.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from "react";
+
+import "./SearchPwd.styles.scss";
+
+class SearchPwd extends Component {
+  constructor() {
+    super();
+    this.state = {
+      userName: "",
+      userId: "",
+      userEmail: "",
+    };
+  }
+
+  handleNameIdEmail = (e) => {
+    this.setState({ [e.target.name]: e.target.value });
+  };
+
+  render() {
+    return (
+      <div className="SearchPwd">
+        <div className="search-pwd-container">
+          <h3 className="search-pwd-title">비밀번호 찾기</h3>
+          <form className="search-pwd-form">
+            <strong>이름</strong>
+            <input
+              className="search-pwd-input user-name"
+              name="userName"
+              type="text"
+              onChange={this.handleNameIdEmail}
+            />
+            <strong>아이디</strong>
+            <input
+              className="search-pwd-input user-id"
+              name="userId"
+              type="text"
+              onChange={this.handleNameIdEmail}
+            />
+            <strong>이메일</strong>
+            <input
+              className="search-pwd-input user-email"
+              name="userEmail"
+              type="text"
+              onChange={this.handleNameIdEmail}
+            />
+            <button className="search-pwd-button">찾기</button>
```

Comment:
```suggestion
            <button type="button" className="search-pwd-button">찾기</button>
```
form 태그의 submit 기능을 활용할 것이 아니라면 button 태그에 type button 값을 부여해주세요. ([링크](https://developer.mozilla.org/ko/docs/Web/HTML/Element/button))

Reply:
onSubmit기능을 활용하고싶어 type = "button"은 따로 주지 않았습니다.
다만 엔터기능은 그대로 활용하고 싶어서 handleSubmit이라는 기능을 구현해주었습니다.